src/account/NexusSmartAccount.ts

```diff
@@ -1039,9 +1039,17 @@ export class NexusSmartAccount extends BaseSmartContractAccount {
   async sendUserOp({
     signature,
     ...userOpWithoutSignature
-  }: Partial<UserOperationStruct>): Promise<UserOpResponse> {
-    const userOperation = await this.signUserOp(userOpWithoutSignature)
-    return await this.sendSignedUserOp(userOperation)
+  }: Partial<UserOperationStruct>, userOpSignature?: Hex): Promise<UserOpResponse> {
+    if (!userOpSignature) {
```

Comment:
how is this change useful?

Reply:
It was supposed to skip the "signUserOp" step which gets signature from module instance, a nice usecase for this was for multi signature user ops where the app needs to first get all signatures and encode them into 1 final signature which will then pass here to override the default behaviour of taking sig from module. Will remove this for now.